.pre-commit-config.yaml

```diff
@@ -37,3 +35,14 @@ repos:
     -   id: blacken-docs
         additional_dependencies: [black==19.10b0]
         exclude: ^.github/
+-   repo: https://github.com/myint/rstcheck
+    rev: ''
```

Comment:
If there is no preferred revision, why not remove the argument? Otherwise, maybe they support a 'stable' argument?

Reply:
Sadly https://github.com/myint/rstcheck doesn't yet have a release with the hook YAML file present, this seemed the best option. See https://github.com/myint/rstcheck/issues/69